src/pages/Dashboard/DashboardV2.css

```diff
@@ -20,7 +20,6 @@
 .token-table-wrapper {
   border-radius: 0.4rem;
   background: #17182c;
-  padding: 0 1.5rem;
```

Comment:
why was this nesessary?

Reply:
in production this rule is overrided by AppCard rule in App.scss. I changed the order of css imports in this PR somehow. So this line should be just removed

Here's the subtask regarding it https://app.asana.com/0/0/1207061230803208/f